### PR TITLE
Feat: Add bring your own cluster

### PR DESCRIPTION
**Pull request by jyash97**
### Dont Merge

### CODE REVIEW

File: src/pages/ClusterPage/info.js [hunk 1]
```diff
@@ -342,6 +342,7 @@ export default class Clusters extends Component {
 		const { showOverlay, isPaid } = this.state;
 
 		const isViewer = this.state.cluster.user_role === 'viewer';
+		const isExternalCluster = this.state.cluster.pricing_plan.startsWith('hosted-arc');
```
Comment (jyash97) [inline src/pages/ClusterPage/info.js]: `TODO:` Change this to check for recipe key in cluster body.

File: src/utils/index.js [hunk 2]
```diff
@@ -20,16 +20,16 @@ export async function getUser() {
...
-	try {
-		const metricsResponse = await fetch(`${ACC_API}/user/metrics`, { credentials: 'include' });
-		if (metricsResponse.status >= 400) {
-			throw new Error();
-		}
-		metrics = await metricsResponse.json();
-	} catch (e) {
-		console.error('Unable to fetch Metrics');
-	}
+	const metrics = '';
```
Comment (jyash97) [inline src/utils/index.js]: `TODO: ` Uncomment this.

File: src/pages/ClusterPage/components/ArcDetailModal.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+					title="Update Cluster Settings"
+					visible={open}
+					okButtonProps={{
+						loading,
+					}}
+					okText="Update Details"
+					onOk={this.handleUpdate}
+					onCancel={this.handleModal}
+				>
+					<h4 className={titleStyle}>Cluster Name:</h4>
```
Comment (jyash97) [inline src/pages/ClusterPage/components/ArcDetailModal.js]: `TODO` Remove this and add URL over here

**Review by siddharthlatest — APPROVED** [review]
lgtm (y)

**Pull request merged**